client/src/modules/purchases/list/list.html

```diff
@@ -55,15 +55,16 @@
           ng-click="PurchaseListCtrl.search()"
           data-method="search"
           class="btn btn-default">
-          <span class="hidden-xs"></span> <span translate>FORM.BUTTONS.SEARCH</span>
+          <span class="fa fa-search"></span> <span class="hidden-xs" translate>FORM.BUTTONS.SEARCH</span>
         </button>
       </div>
     </div>
   </div>
 </div>
 
-<div class="flex-util bh-filter-bar">
+<div class="flex-util" style="min-height : 35px; padding-top : 7px; max-height: initial">
   <bh-filters
+    style="max-width:90%"
```

Comment:
I thought it was a fix to a problem as it is in a file recently reviewed. I will remove it!